MobileCenter/MobileCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -40,46 +40,43 @@ -(void)initTables {
 
 #pragma mark - Public
 
-- (BOOL)saveLog:(id <MSLog>)log withStorageKey:(NSString *)storageKey {
+- (BOOL)saveLog:(id<MSLog>)log withGroupID:(NSString *)groupID {
   if (!log) {
     return NO;
   }
   NSData *logData = [NSKeyedArchiver archivedDataWithRootObject:log];
   NSString *base64Data = [logData base64EncodedStringWithOptions:NSDataBase64EncodingEndLineWithLineFeed];
   NSString *addLogQuery = [NSString stringWithFormat:@"insert or replace into %@ values ('%@', '%@')",
-                                   kMSLogTableName, storageKey, base64Data];
+                           kMSLogTableName, groupID, base64Data];
   return [self.connection executeQuery:addLogQuery];
 }
 
-- (NSArray <MSLog> *)deleteLogsForStorageKey:(NSString *)storageKey {
-  NSArray<MSLog> *logs = [self getLogsWith:storageKey];
-  [self deleteLogsWith:storageKey];
+- (NSArray<MSLog> *)deleteLogsForGroupID:(NSString *)groupID {
+  NSArray<MSLog> *logs = [self getLogsWith:groupID];
+  [self deleteLogsWith:groupID];
   return logs;
 }
 
-- (void)deleteLogsForId:(NSString *)logsId withStorageKey:(NSString *)storageKey {
+- (void)deleteLogsForId:(NSString *)logsId withGroupID:(NSString *)groupID {
 
   // FIXME: logsId ?
-  [self deleteLogsWith:storageKey];
+  [self deleteLogsWith:groupID];
 }
 
-- (BOOL)loadLogsForStorageKey:(NSString *)storageKey withCompletion:(nullable MSLoadDataCompletionBlock)completion {
-  NSArray<MSLog> *logs = [self getLogsWith:storageKey];
-
+- (BOOL)loadLogsForGroupID:(NSString *)groupID withCompletion:(nullable MSLoadDataCompletionBlock)completion {
+  NSArray<MSLog> *logs = [self getLogsWith:groupID];
   if (completion) {
-
-    // FIXME: batchId ?
     completion(logs.count > 0, logs, @"");
   }
-
   return logs.count > 0;
 }
 
-- (void)closeBatchWithStorageKey:(NSString *)storageKey {
-  // TODO:
+- (void)closeBatchWithGroupID:(NSString *)groupID {
```

Comment:
`closeBatchWithGroupID` was the workaround for file storage to close any file when channel's batch got full. So that next time it's picked up it'll not retrieve more than the number of expected logs. To quickly fix this we should add a log limit to the `- (BOOL)loadLogsForGroupID:(NSString *)groupID withCompletion:(nullable MSLoadDataCompletionBlock)completion` method, that way the channel, while loading logs, will never exceed it's batch max number of logs ;).